portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,55 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.comment.CommentStorage;
+import com.google.sps.config.Constants;
+import com.google.appengine.api.datastore.Entity;
+import java.util.ArrayList;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content.*/
```

Comment:
Consider updating this comment with the update that this now handles comments data.